### ISSUE TITLE
IOS_VERSION returns major version and not major.minor.patch

### ISSUE THREAD

**Issue by forbesjo** [videojs/video.js#4917]
## Description
`videojs.browser.IOS_VERSION` returns only the major iOS version and not the full version as I expected.

I have worked around my issue by using feature detection instead but it would be worth documenting this API's behavior.

## Steps to reproduce
Explain in detail the exact steps necessary to reproduce the issue.
1. On mobile Safari run `videojs.browsers.IOS_VERSION`

### Expected
The string `'11.2.0'` or the number `11.2` (similar to `ANDROID_VERSION`)

### Actual
The string `'11'`

**Comment by gkatsev** [comment]
Yup, this is far from ideal. The browser module we have is a bit messy and needs some cleanup.
